### ISSUE TITLE
BarcodeScanner is not working in a modal component

### ISSUE THREAD

**Issue by masrc** [EddyVerbruggen/nativescript-barcodescanner#127]
Hey! I have an app that need to use the BarcodeScanner, the component that's using BarcodeScanner is called twice and in one of this calls, is a modal component. In runtime this works perfectly, but running with webpack the modal component is not oppening and return the follow error: "Token must be defined!". Do you have any idea how to fix this?

**Comment by EddyVerbruggen** [comment]
What token? And which platform(s)?

And how do you instantiate the barcode?

**Comment by bnbs** [comment]
I don't know what token is this. When I use this plugin with webpack this error occurs and without webpack or without the plugin the error doesn't occurs. We're using android platform and we followed [this](https://www.thepolyglotdeveloper.com/2016/03/implement-a-barcode-scanner-using-telerik-nativescript/) tutorial and we added [this]( https://github.com/EddyVerbruggen/nativescript-barcodescanner#webpack-usage ) code too.
